.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this from commit. You can also add gitignore for that.

Reply:
Done

Reply:
I meant the whole file.